x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/geogrid.tsx

```diff
@@ -0,0 +1,246 @@
+            }
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <TargetField
+            helpText={
+              <FormattedMessage
+                id="xpack.ingestPipelines.pipelineEditor.geoGrid.targetHelpText"
+                defaultMessage="The field to assign the polygon shape to. By default {field} is updated in-place."
+                values={{ field: <EuiCode>{'field'}</EuiCode> }}
+              />
+            }
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <EuiSpacer size="m" />
+
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <UseField
+            componentProps={{
+              euiFieldProps: {
+                'data-test-subj': 'tileTypeField',
+                options: [
+                  {
+                    value: 'geohash',
+                    text: i18n.translate(
+                      'xpack.ingestPipelines.pipelineEditor.geoGrid.geohashOption',
+                      { defaultMessage: 'Geo-Hash' }
+                    ),
+                  },
+                  {
+                    value: 'geotile',
+                    text: i18n.translate(
+                      'xpack.ingestPipelines.pipelineEditor.geoGrid.geotileOption',
+                      { defaultMessage: 'Geo-Tile' }
+                    ),
+                  },
+                  {
+                    value: 'geohex',
+                    text: i18n.translate(
+                      'xpack.ingestPipelines.pipelineEditor.geoGrid.geohexOption',
+                      { defaultMessage: 'Geo-Hex' }
+                    ),
+                  },
+                ],
+              },
+            }}
+            config={fieldsConfig.tileType}
+            component={SelectField}
+            path="fields.tile_type"
+          />
+        </EuiFlexItem>
+        <EuiFlexItem>
+          <UseField
+            componentProps={{
+              euiFieldProps: {
+                'data-test-subj': 'targetFormatField',
+                options: [
+                  {
+                    value: 'GeoJSON',
+                    text: i18n.translate(
+                      'xpack.ingestPipelines.pipelineEditor.geoGrid.geojsonOption',
+                      { defaultMessage: 'GeoJSON' }
+                    ),
+                  },
+                  {
+                    value: 'WKT',
+                    text: i18n.translate('xpack.ingestPipelines.pipelineEditor.geoGrid.wktOption', {
+                      defaultMessage: 'WKT',
+                    }),
+                  },
+                ],
+              },
+            }}
+            config={fieldsConfig.target_format}
+            component={SelectField}
+            path="fields.target_format"
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+
+      <EuiSpacer size="m" />
+
+      <UseField
+        data-test-subj="parentField"
+        config={fieldsConfig.parent}
+        component={Field}
+        path="fields.parent"
```

Comment:
`fields.parent_field`
Same comment for the rest of the paths where the field name needs to be updated.